app/controllers/simplified-account/settings/index.controller.js

```diff
@@ -0,0 +1,20 @@
+const formatSimplifiedAccountPathsFor = require('../../../utils/simplified-account/format/format-simplified-account-paths-for')
+const paths = require('../../../paths')
+const { LIVE } = require('../../../models/go-live-stage')
+
+function get (req, res) {
+  const account = req.account
+  const service = req.service
+  // the default setting for the index view is dependent on the account type and go live state
+  if (account.type === 'test' && service.currentGoLiveStage !== LIVE) {
```

Comment:
Here we've got an `enum` (probably just a basic object acting as an enum) being used to check the go-live status against and a string literal for the account type.  I'm wondering whether we should introduce an `enum` (just a basic object acting as an enum) for test and live, that's for two reasons:

 - To track usage across the app
 - Just in case the value changes at some point in the future